samples/packaged/CopilotChat/SKWebApi/Program.cs

```
@@ -17,6 +18,14 @@ public static void Main(string[] args)
 
         builder.Host.ConfigureAppSettings();
 
+        // Set port to run on
+        string serverPortString = builder.Configuration.GetSection("ServicePort").Get<string>();
+        if (!int.TryParse(serverPortString, out int serverPort))
```

Should we set a default serverPort or fail if one is not found?